core/src/main/java/hudson/Functions.java

```diff
@@ -681,11 +681,22 @@ public static void configureAutoRefresh(HttpServletRequest request, HttpServletR
         /* feature has been removed */
     }
 
+    /**
+     * @deprecated removed without replacement
+     */
     @Deprecated
     public static boolean isAutoRefresh(HttpServletRequest request) {
```

Comment:
This adds a new method, right? Should probably just be dropped instead. Keep the existing signature around, but it's entirely unused and there should not be callers left.

Reply:
I ended up abandoning this approach entirely for now.